app/src/main/res/layout/exploration_activity.xml

```diff
@@ -31,7 +31,8 @@
         android:background="@color/component_color_shared_secondary_toolbar_color"
         android:minHeight="?attr/actionBarSize"
         android:theme="@style/Widget.AppCompat.ActionBar"
-        app:navigationIcon="@drawable/ic_close_white_24dp">
+        app:navigationIcon="@drawable/ic_close_white_24dp"
+        app:navigationContentDescription="@string/hints_andSolution_close_icon_description">
```

Comment:
If you want it to read as close, that's fine, but reusing the string that is named after a different screen is not advisable. You should create a new string for it named after the file. Please open `strings.xml` and search for "close" so that you can see examples.

Reply:
Also note that we add tests for these content descriptions, so changing it here will definitely cause failures in ExplorationActivityTest. Please see [wiki](https://github.com/oppia/oppia-android/wiki/Interpreting-CI-Results) for reference on intepreting CI failures, in order to know which tests you need to update.

Reply:
Ok @adhiamboperes,
For this Pr i will just make the necessary chages asked in the issue.